utils/countryUtils.js

```diff
@@ -1,39 +1,32 @@
 const stringUtils = require('./stringUtils');
 const countryData = require('./countries');
 
-/**
- * Transform null values to 0 (DEEPLY TRANSFORM)
- * @param {Object} object country/continent object
- * @returns  {Object} 		transformed object
- */
-const transformNull = (object) => {
-	if (typeof object !== 'object') {
-		return object;
+
+const convertNullToZero = (continentOrCountry = null) => {
+
+	const COUNTRY_INFO = 'countryInfo';
+
+	if (typeof continentOrCountry !== "object" || !continentOrCountry) {
+
+		throw new TypeError(`${continentOrCountry}` + "should be a an object with non null value")
+
+	}
+	if (Object.keys(continentOrCountry).length === 0) {
+		throw new Error(`${continentOrCountry}` + "should not be an empty object")
```

Comment:
it does not matter if its an empty object, so we dont need to throw an error if it is

Reply:
To improve Developer Experience ( DX ).

When you design an API for developers handling errors matter.

May you see it not matter, but with an empty `object` the function will not return an output. I mean the empty object still an empty object.

Reply:
an empty object is in no way an error that needs to be handled

Reply:
Yes, you're right it's not an explicit error but it's a silent bug.